lib/devices/ip.ml

```diff
@@ -85,7 +85,7 @@ type ipv6_config = {
 (** Types for IP manual configuration. *)
 
 let ipv4_qubes_conf =
-  let packages = [ package ~min:"0.9.0" ~max:"0.10.0" "mirage-qubes-ipv4" ] in
+  let packages = [ package ~min:"0.10.0" ~max:"0.11.0" "mirage-qubes-ipv4" ] in
```

Comment:
```suggestion
  let packages = [ package ~min:"0.9.0" ~max:"0.11.0" "mirage-qubes-ipv4" ] in
```

The 0.9 release is fine as well, we don't need to update the lower bound, only allow the upper bound.